README.md

```diff
@@ -346,10 +74,19 @@ These are frontend options.
       --EmitONNXIR    - Ingest ONNX and emit corresponding ONNX dialect.
       --EmitMLIR      - Lower model to MLIR built-in transformation dialect.
       --EmitLLVMIR    - Lower model to LLVM IR (LLVM dialect).
-      --EmitLib       - Lower model to LLVM IR, emit (to file) LLVM bitcode for model, compile and link it to a shared library.
+      --EmitLib       - Lower model to LLVM IR, emit (to file) LLVM bitcode for model,            compile and link it to a shared library.
```

Comment:
Remove extra white space.

Reply:
end of line was missing, made also output more compact.